tests/test_rooms/test_daily_room_events.py

```diff
@@ -22,9 +28,39 @@ def test_analytics_for_daily_events(self, mock_get_json):
         )
 
     def test_analytics_for_daily_events_wrong_format_date(self, mock_get_json):
+        """
+            Test querying analytics for daily events with
```

Comment:
What does this test try to find out? same to line 44 and 58. Kindly describe what the tests are trying to achieve

Reply:
This has been worked on @Bonifase